### PR TITLE
Admin sees current winning bid in header

### LINKED ISSUE #1289: Admin messaging - Auction show page header - bid status on open auctions
As an admin, I want to be aware of the bid status of any auction at any given time, so I should see the current bid status in the header of the show page

- [x] **Scenario: Admin sees current low bid in header of auction show page for open auctions**
- Given I am an authenticated admin
- And there is an open auction
- And the auction has bids
- When I go to the auction show page (https://micropurchase.18f.gov/admin/auctions/[id])
- Then I should see a section labeled "Current low bid:"
- And I should see the current lowest bid in that section
- And I should see the name of the vendor who made that bid in parentheses

## Notes
Use the styles referenced below (or as close as you can get)

screen shot 2016-09-20 at 4 45 37 pm

### PR DESCRIPTION
**Pull request by harrisj**
**MERGE #1353 first**

Fixes #1289

### CODE REVIEW

**Comment by jessieay** [comment]
Merge conflicts 😢

**Comment by harrisj** [comment]
Let's wait until #1358 is merged

**Comment by harrisj** [comment]
Rebased

File: app/views/admin/auctions/show.html.erb [hunk 1]
```diff
@@ -9,8 +9,6 @@
             <%= render partial: @view_model.csv_report_partial,
               locals: { auction: @view_model } %>
           </div>
-        </div>
-        <div class="auction-subtitles">
```
Comment (jessieay) [inline app/views/admin/auctions/show.html.erb]: did you mean to remove this?

File: app/presenters/bidding_status_presenter/base.rb [hunk 2]
```diff
@@ -25,6 +25,10 @@ def label
     ''
   end
 
+  def bid_label(_user)
```
Comment (jessieay) [inline app/presenters/bidding_status_presenter/base.rb]: do we need this? I am not sure what we are using it for here

Comment (harrisj) [inline app/presenters/bidding_status_presenter/base.rb]: Oh, I was planning ahead. I removed it

**Pull request merged**